nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/PutMongo.java

```diff
@@ -112,8 +112,10 @@
         .allowableValues(UPDATE_WITH_DOC, UPDATE_WITH_OPERATORS)
         .defaultValue(UPDATE_WITH_DOC.getValue())
         .description("Choose an update mode. You can either supply a JSON document to use as a direct replacement " +
-                "or specify a document that contains update operators like $set and $unset")
-        .build();
+                "or specify a document that contains update operators like $set, $unset, and $inc."+
+                        "When Operators mode is enabled, the flowfile content is expected to be the operator part"+
+                        "for example: {$set:{\"key\": \"value\"},$inc:{\"count\":1234}} and Update query has to be the record to be updated")
```

Comment:
The part here after `and` is somewhat ambiguous and could potentially confuse people doing upserts. Other than that, it looks fine.

Reply:
I'd suggest we strike that part and replace it with something like this:

> and the update query will come from the configured `Update Query` property.

If that works for you, I can make the change myself.

Reply:
@MikeThomsen yes please go ahead with the changes.

Reply:
Can you git cherry-pick this onto a branch named NIFI-5589? It's currently sitting on your master.

Reply:
(Just push that branch to GitHub, I'll fetch it and merge)